OpenRA.Game/ExternalMods.cs

```diff
@@ -59,7 +59,8 @@ Sheet CreateSheet()
 
 		public ExternalMods()
 		{
-			sheetBuilder = new SheetBuilder(SheetType.BGRA, CreateSheet);
+			if (Game.Renderer != null)
```

Comment:
IMO it would be good to add a comment here, something like `// Don't try to load mod icons if we don't have a texture to put them in`?

Reply:
Done.